pkg/morph/client/notifications.go

```diff
@@ -14,7 +14,12 @@ func (c *Client) Close() {
 	// closing should be done via the channel
 	// to prevent switching to another RPC node
 	// in the notification loop
-	c.closeChan <- struct{}{}
+	select {
+	case c.closeChan <- struct{}{}:
```

Comment:
is it possible to just close the chan here? do not remember why that was done that way but now closing looks better to me

Reply:
Sure, replaced with close.